include/nccl_ofi_cuda.h

```diff
@@ -25,21 +37,80 @@ int nccl_net_ofi_cuda_init(void);
  * @return	0 on success
  *		non-zero on error
  */
-int nccl_net_ofi_get_cuda_device(void *data, int *dev_id);
+static inline int nccl_net_ofi_get_cuda_device(void *data, int *dev_id)
```

Comment:
why static inline?  This seems like overkill.

I see no benefit to making this file includable with the Neuron build; it just adds complexity better handled elsewhere.